orchestra/bots/tests/test_views.py

```diff
@@ -17,7 +17,12 @@ class StaffBotViewTest(OrchestraTestCase):
     def setUp(self):
         super().setUp()
         setup_models(self)
-        self.request_client = RequestClient()
+        self.worker = self.workers[0]
+        self.worker.user.is_superuser = True
+        self.worker.user.save()
```

Comment:
add a test to verify that an unauthorized user receives a json error message (status code should be 200 so slack can display the message nicely though)